ios/Classes/SwiftLiveActivitiesPlugin.swift

```diff
@@ -1,91 +1,97 @@
+import ActivityKit
 import Flutter
 import UIKit
-import ActivityKit
 
 public class SwiftLiveActivitiesPlugin: NSObject, FlutterPlugin, FlutterStreamHandler {
   private var urlSchemeSink: FlutterEventSink?
   private var appGroupId: String?
   private var sharedDefault: UserDefaults?
-  
+  private var appLifecycleLifeActiviyIds = [String]()
+
   public static func register(with registrar: FlutterPluginRegistrar) {
     let channel = FlutterMethodChannel(name: "live_activities", binaryMessenger: registrar.messenger())
     let urlSchemeChannel = FlutterEventChannel(name: "live_activities/url_scheme", binaryMessenger: registrar.messenger())
     let instance = SwiftLiveActivitiesPlugin()
     registrar.addMethodCallDelegate(instance, channel: channel)
     urlSchemeChannel.setStreamHandler(instance)
-    
+
     registrar.addApplicationDelegate(instance)
   }
-  
+
   public func onListen(withArguments arguments: Any?, eventSink events: @escaping FlutterEventSink) -> FlutterError? {
     urlSchemeSink = events
     return nil
   }
-  
+
   public func onCancel(withArguments arguments: Any?) -> FlutterError? {
     urlSchemeSink = nil
     return nil
   }
-  
+
   private func initializationGuard(result: @escaping FlutterResult) {
-    if (self.appGroupId == nil || self.sharedDefault == nil) {
+    if self.appGroupId == nil || self.sharedDefault == nil {
       result(FlutterError(code: "NEED_INIT", message: "you need to run 'init()' first with app group id to create live activity", details: nil))
     }
   }
 
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
     if #available(iOS 16.1, *) {
-      switch (call.method) {
+        
+      switch call.method {
```

Comment:
What are you using to format your swift code? I had problems getting the formatting right when editing the files in xcode 🤔

Reply:
I notice that "Indent switch/case labels" was not active on my Xcode for Swift...
I enabled it 👍

Reply:
No i usually just go with "Structure" -> "Reindent". I did that and pushed the changes. The diff is huge, but I didn't added anything else besides the change mentioned in the other discussion 👍🏽